README.md

```diff
@@ -1,49 +1,48 @@
 # blue-harvest
 
-## Install
+## Getting Started
 You can npm to install the package:
-`npm install blue-harvest --save-dev`,
-or if you are using Yarn,
-`yarn add blue-harvest -D`
+`npm install https://github.com/copperleaftech/blue-harvest`.
 
 ## Writing Tests
 ### Configurations
-In order for you to use `async await` in the tests, the following line
-```javascript
-SELENIUM_PROMISE_MANAGER: false;
-```
- should be added to the `protractor.conf.js` file.
+In order for you to use `async await` in the tests, `SELENIUM_PROMISE_MANAGER: false` should be added to the `protractor.conf.js` file.
 
 To regulate the resolution of the screenshots, add 
-```javascript
-onPrepare() {
-    browser.driver.manage().window().setSize(1366, 1024);
-    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
-  }
-```
-in `protractor.conf.js`, inside the `config` block.
+`browser.driver.manage().window().setSize(1366, 1024)`
+ inside `onPrepare()`.
 ### Comparing Screenshots
 To test if a page has changed from a previous screenshot:
 ```
 let result =  await blueharvest.compareScreenshot(
-                   await browser.takeScreenShot(), pathToGolden);
+                   await browser.takeScreenShot(), 'reference.png', 'root/errors');
 ```
-The difference between the two screenshots will be highlighted in pink.
+The difference between the two screenshots will be highlighted in pink. The third parameter defines where to save the difference images and is optional. If the destination path
+for the created difference image is not given but the test fails, `compareScreenshot` will still reject the promise, but will not save the difference image. 
```

Comment:
If the destination path is not provided and the test fails, should we instead create a default directory? This behavior sounds not right.

Reply:
I can imagine situations where saving diff image is not necessary for some testing purposes. Plus if the user really wants to collect the images, I'd imagine that saving to a designated folder makes more sense.

Reply:
The rationale behind our decision was that, under a git repository (specially when you are running e2e tests locally) we'd want the failed comparison images to be saved to an untracked folder for convenience (just so no one checks them in). I think that if you want them saved you should tell it where to save it too.  For CI it also makes sense to provide where you want to save the failed diff images.   I can see your point of having a default directory too maybe, it would be better to save them to a temp folder and the message will reflect where they are saved.

In the near future we will either move the screenshot functionality to a separate repo or find an existing one that could do the job.